packages/loader/container-loader/src/container.ts

```diff
@@ -820,6 +820,10 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         const appSummary: ISummaryTree = this.context.createSummary();
         const protocolSummary = this.captureProtocolSummary();
         const combinedSummary = combineAppAndProtocolSummary(appSummary, protocolSummary);
+
+        if (this.loader.services.detachedBlobStorage && this.loader.services.detachedBlobStorage.size > 0) {
```

Comment:
Curious if you tried the optional-chaining operator ('?.size') and ran into the ESLint strict bool rule?

(No strong preference. You can disregard if you like &&.)